vitelabs/vuilder
# subcommand test and node

support subcommands: test and node

upgrade:

1. remove old version and install latest version
npm remove @vite/vuilder
npm install https://github.com/vitelabs/vuilder.git#feat/subcommand --save-dev

2. upgrade code
see: https://github.com/vitelabs/dapp-buymeacoffee/commit/20c2bf0e1f9e751a8478f7a068424433e80686a4
tips: 
remove `import { describe } from "mocha";` from *.spec.ts
add `defaultNode` in `test/vite.node.json`

3. run command:
`npx vuilder test --config test/vite.node.json`  run test with special node
`npx vuilder test`    run test with default node
`npx vuilder test test/*.spec.ts`  run test for test/*.spec.ts
`npx vuilder node`  run a vite node
`npx vuilder compile xxx.solpp` compile solpp in contracts  directory
